c2corg_api/models/waypoint.py

```diff
@@ -139,7 +139,7 @@ class _WaypointMixin(object):
     phone_custodian = Column(String(50))
 
     # gardiennage (gite, camp_site, hut, abri, bivouac, base_camp)
-    custodianship = Column(Boolean)
+    custodianship = Column(enums.custodianship_type)
```

Comment:
With 'multi-value' you don't mean you can select multiple values? You would have to use `Column(ArrayOfEnum(enums.custodianship_type))` in that case.